dashboard/src/components/Tabs/Builds/BuildCards.tsx

```diff
@@ -35,13 +33,14 @@ const ErrorsSummaryBuild = ({
         <DumbSummary summaryHeaders={summaryHeaders}>
           {summaryBody?.map(row => {
             return (
-              <SummaryItem
+              <MemoizedSummaryItem
                 key={row.arch.text}
                 arch={{ text: row.arch.text }}
+                diffFilter={diffFilter}
                 onClickCompiler={value =>
-                  toggleFilterBySection(value, 'compilers')
+                  toggleFilterBySection(value, 'compilers' as T)
                 }
-                onClickKey={value => toggleFilterBySection(value, 'archs')}
+                onClickKey={value => toggleFilterBySection(value, 'archs' as T)}
```

Comment:
please avoid `as` as much as possible, `as` can break an app since it creates a vulnerability